Base/usr/share/man/man7/setuid_overview.md

```diff
@@ -0,0 +1,71 @@
+## Name
+
+Overview of real, effective, and saved user and group ids
+
+## Description
+
+Each process runs as user and a group. By default, a process started by a user has its user id set to that user and its group id set to that user's primary group.
+
+Each file has permissions that control if it can be read, written, and executed by its owner, other users in the owner's group, and by others. A process has the same access permissions as its user and its group.
+
+In addition to the access permissions bits, executables may have the "Set User ID"  and the "Set Group ID" bit set. Executables with these bits set run with the user or group id of the executable, instead of with the user and group id of the current user. These binaries are also called "SUID binaries" or "setuid binaries" (or "SGID binaries" or "setgid binaries" for the "Set Group ID" bit).
+
+The motivation behind SUID binaries is that they allow users to do tasks that would normally require elevated permissions, without having to give users these permissions fully.
+
+For example, `/usr/ping` has the Set User ID bit set and is owned by user root in group root. So if any user executes `/usr/ping`, it will run as root, which means it will be able to send network packets, even if the current user doesn't normally have network access.
+
+For another example, `passwd` changes the password of the current user, which means it has to write to `/etc/shadow` -- but the current user is not supposed to have write access to `/etc/shadow` so that they cannot change passwords of other users. The solution is to make `passwd` a SUID binary. (SerenityOS currently doesn't have support for user passwords; this examples is more convincing on other UNIXes.)
```

Comment:
Do we have actually `passwd`(1)?

Ah, I see you even mention that we don't; then you should start this with "many other Unix systems also include a `passwd` utility..." or something like that, instead of simply mentioning it at the end.

Reply:
Done.